packages/plugins/embarkjs/package.json

```diff
@@ -21,7 +21,7 @@
   ],
   "license": "MIT",
   "repository": {
-    "directory": "packages/stack/embarkjs",
+    "directory": "packages/plugins/embarkjs",
```

Comment:
`"homepage"` should be updated likewise:

https://github.com/embarklabs/embark/blob/71d1650e5f13b848caed1d4e15060d09aa85b797/packages/plugins/embarkjs/package.json#L7